drivers/gpu/drm/i915/i915_drv.c

```diff
@@ -1616,11 +1616,13 @@ static int i915_driver_init_hw(struct drm_i915_private *dev_priv)
  */
 static void i915_driver_cleanup_hw(struct drm_i915_private *dev_priv)
 {
-#ifdef __linux__
 	struct pci_dev *pdev = dev_priv->drm.pdev;
```

Comment:
Add `#if defined(pci_disable_msi)` here like below to fix the following:
```c
drivers/gpu/drm/i915/i915_drv.c:1619:18: error: unused variable 'pdev' [-Werror,-Wunused-variable]
        struct pci_dev *pdev = dev_priv->drm.pdev;
                        ^
```

Reply:
argh, werror. Fixed